### PR TITLE
Add ATLAS_PK documentation

### PR DESCRIPTION
**Pull request by pblottiere**

### CODE REVIEW

File: source/docs/user_manual/working_with_ogc/server/services.rst [hunk 1]
```diff
@@ -689,6 +691,19 @@ If the ``FORMAT`` parameter is different from one of these values, then an
 exception is returned.
 
 
+
+ATLAS_PK
+^^^^^^^^
+
+This parameter allows activation of Atlas rendering by indicating which
+features we want to print. In order to retrieve an atlas with all features,
+the ``*`` symbol may be used (according to the maximum number of feature
```
Comment (havatv) [inline source/docs/user_manual/working_with_ogc/server/services.rst]:
```suggestion
the ``*`` symbol may be used (according to the maximum number of features
```

**Comment by ghtmtt** [comment]
@havatv why have to to wait for the merge? This docs are going straight on testing docs, so no matters the QGIS version (always the last one).

We create different branch only for LTR docs

**Comment by havatv** [comment]
@ghtmtt, yes.  As far as I have understood, these docs will become LTR (3.4) docs, and we must wait with the 3.6 updates until the LTR is official (when 3.6 is released), and the 3.4 branch has been created.

**Comment by havatv** [comment]
Yes. That is one reason why we try to avoid 3.6 pull requests currently (in addition to the urgency of getting 3.4 documenation in place before it becomes LTR).

File: source/docs/user_manual/working_with_ogc/server/services.rst [hunk 2]
```diff
@@ -689,6 +691,19 @@ If the ``FORMAT`` parameter is different from one of these values, then an
...
+allowed in the project configuration).
+
+When ``FORMAT`` is ``pdf``, a PDF document with one feature per page will be
+returned. For all other formats, only the first page is returned.
```
Comment (DelazJ) [inline source/docs/user_manual/working_with_ogc/server/services.rst]: i'm confused by this behavior so, updating my comment in https://github.com/qgis/QGIS-Documentation/pull/3345#discussion_r246736836:
 
1. _For all other formats, only the **first page** is returned._
Is that possible to have a two pages layout on server? If yes, with jpg, I'd lose the second page? Am I right?

2. _"When FORMAT is pdf, a PDF document with **one feature per page** is returned."_
For me (i'm not server user but it'd be inline with the desktop) it's more a matter of file(s) per feature than page per feature. 

What I naively thought would be the behavior is:
`When FORMAT is pdf, a single PDF document combining each feature page(s) is returned. For all other formats, each feature is returned in separate file(s) per page.`

The limitation with pdf on server would be that I can not output separate pdf files for each feature?

**Comment by ghtmtt** [comment]
@DelazJ @havatv can we merge this PR? 3.4 bis branched and this will not be backported.

**Comment by DelazJ** [comment]
@ghtmtt I had some questions regarding the description that did not get answer yet.

**Comment by havatv** [comment]
I also had a trivial suggestion that could be accepted.

**Comment by SrNetoChan** [comment]
19 days have passed since the last message, should we take it from here?

**Comment by DelazJ** [comment]
@SrNetoChan I have an unreplied comment based on my misunderstanding of a description which imo is inconsistent with (/opposite to) the desktop behavior. The current description gives me another interpretation. It could be nice we get this sorted before merging unless it's considered as falling within the facultative review comments (/me not convinced though)

**Comment by pblottiere** [comment]
Hi @DelazJ and @havatv,

Sorry for not updating my branch. To be honest, I totally forgot this PR...

I can assure you that I'll take a look tomorrow!

**Comment by DelazJ** [comment]
Hi @pblottiere No problem you know. It's our "fault" if the PR had to wait for merging, so it's normal if meanwhile you forgot. Anyway, thanks for the great work you are doing

File: source/docs/user_manual/working_with_ogc/server/services.rst (hunk 2 above)
Comment (pblottiere) [reply source/docs/user_manual/working_with_ogc/server/services.rst]: Hi @DelazJ,

> 1. For all other formats, only the first page is returned.
> Is that possible to have a two pages layout on server? If yes, with jpg, I'd lose the second page? Am I right?

Yes and yes :).

> 2. "When FORMAT is pdf, a PDF document with one feature per page is returned."
> For me (i'm not server user but it'd be inline with the desktop) it's more a matter of file(s) per feature than page per feature.

> The limitation with pdf on server would be that I can not output separate pdf files for each feature?

Actually, the server is able to return only one document per request. In the case of PDF format, the document may contain several pages.

What do you think about:

"When FORMAT is pdf, a single PDF document combining each feature page(s) is returned. For all other formats, a single page is returned."

File: source/docs/user_manual/working_with_ogc/server/services.rst [hunk 3]
```diff
@@ -858,6 +860,19 @@ If the ``FORMAT`` parameter is different from one of these values, then an
...
+
+ATLAS_PK
+^^^^^^^^
+
+This parameter allows activation of Atlas rendering by indicating which
+features we want to print. In order to retrieve an atlas with all features,
+the ``*`` symbol may be used (according to the maximum number of features
+allowed in the project configuration).
+
+When ``FORMAT`` is ``pdf``, a PDF document with one feature per page will be
```
Comment (havatv) [inline source/docs/user_manual/working_with_ogc/server/services.rst]:
```suggestion
When ``FORMAT`` is ``pdf``, a single PDF document combining the feature pages is returned.
For all other formats, a single page is returned.
```

File: source/docs/user_manual/working_with_ogc/server/services.rst [hunk 4]
```diff
@@ -858,6 +860,19 @@ If the ``FORMAT`` parameter is different from one of these values, then an
...
+returned. For all other formats, only the first page is returned.
```
Comment (havatv) [inline source/docs/user_manual/working_with_ogc/server/services.rst]:
```suggestion
For all other formats, a single page is returned.
```

**Comment by havatv** [comment]
It would be nice to get this merged.  @DelazJ, can you take a look at my latest suggestions?

**Pull request merged**